components/components.go

```diff
@@ -159,18 +160,15 @@ var IsWorkingDirDependant FilterFunc = func(cmp Component) (bool, error) {
 	return false, nil
 }
 
-// IsInstalledFilter filters Components that have its image installed, with
+// IsInstalled filters Components that have its image installed, with
 // the exact version
-var IsInstalledFilter FilterFunc = func(cmp Component) (bool, error) {
-	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
-	defer cancel()
-
-	return docker.IsInstalled(ctx, cmp.Image, cmp.Version)
+var IsInstalled FilterFunc = func(cmp Component) (bool, error) {
+	return cmp.IsInstalled()
 }
 
-// IsRunningFilter filters Components that have a container running, using
+// IsRunning filters Components that have a container running, using
 // its image with the exact version
-var IsRunningFilter FilterFunc = func(cmp Component) (bool, error) {
+var IsRunning FilterFunc = func(cmp Component) (bool, error) {
```

Comment:
Why are these variables instead of standard functions?

Reply:
To ensure that they are `FilterFunc` type, and document they are intended to be used as `FilterFunc`. Is it not idiomatic?

Reply:
not really, no

the more idiomatic way would be to have:

```go
func IsRunning() FilterFunc  {
   return func(cmp Component) (bool, error) { ... }
}
```

Reply:
> the more idiomatic way would be to have:
> 
> ```go
> //  IsRunning is a FilterFunc...
> func IsRunning(cmp Component) (bool, error)  {
>    ...
> }
> ```

I've included that in the squashed changes before merging